src/SourceBuild/patches/runtime/0001-Remove-unnecessary-suppressions.patch

```diff
@@ -1,64 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
@mthalman 
Aren't there 2 patches that need to be removed?

Reply:
https://github.com/dotnet/installer/blob/04244dd8d4ba4409d1fd71a7fabd27d8d7338950/src/SourceBuild/patches/runtime/0002-Update-API-compat-suppressions.patch#L15

Reply:
The second one is still needed as the referenced commit doesn't have that change yet.